hypothesis-python/tests/ghostwriter/test_ghostwriter.py

```diff
@@ -33,8 +36,16 @@ def get_test_function(source_code):
     # Note that this also tests that the module is syntatically-valid,
     # AND free from undefined names, import problems, and so on.
     namespace = {}
-    exec(source_code, namespace)
-    tests = [v for k, v in namespace.items() if k.startswith(("test_", "Test"))]
+    try:
+        exec(source_code, namespace)
+    except SyntaxError:
+        print(source_code)
```

Comment:
If we're going to do this then maybe add a header saying what the print is? Might make more sense to just use `--pdb` for debugging though.